ui/js/actions/content.js

```diff
@@ -342,8 +342,23 @@ export function doPurchaseUri(uri, purchaseModalName) {
     const costInfo = selectCostInfoForUri(state, { uri });
     const { cost } = costInfo;
 
-    // the file is free or we have partially downloaded it
-    if (cost === 0 || (fileInfo && fileInfo.download_directory)) {
+    /**
+     * Play video without a price warning dialog if:
+     *  - The file is free
+     *  - Instant purchase is enabled and the file is cheap enough
+     *  - The file is already downloading
+     */
+    const instantPurchaseEnabled = lbry.getClientSetting(
+      "instantPurchaseEnabled"
+    );
+    const instantPurchaseMax = lbry.getClientSetting("instantPurchaseMax")[
```

Comment:
If we want to make it possible to have the max instant purchase price be denominated in USD, then we'll have to do a currency conversion here. If not, then we'll have to add an option to FormFieldPrice to disable the currency selector and always use LBC (very easy).

Reply:
I believe the app already has the ability to do client side currency conversion, so it shouldn't be too bad.

Reply:
Also, make sure to put these in `const/settings.js`

Reply:
Oh right, you'd think I'd remember that we can do client side currency conversion, given that I wrote the code for it :P Will do.

Reply:
Oh, right. OK, I'll make it handle USD. You'd think I would remember that we have client-side currency conversion, given that I wrote the code for it...